webui/react/src/pages/ModelDetails.tsx

```diff
@@ -228,8 +229,7 @@ const ModelDetails: React.FC = () => {
 
       const newSettings = {
         sortDesc: order === 'descend',
-        /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-        sortKey: columnKey as any,
+        sortKey: isOfSortKey(columnKey) ? columnKey : V1GetModelVersionsRequestSortBy.UNSPECIFIED,
```

Comment:
i dont like the way to handle type like that, but i cannot change the type of `columnKey` cuz it comes from Antd.

Reply:
What don't you like about it?

Reply:
`isOfSortKey` has `include`, so it's not O(1) and more code. I know that its trivial but I just feel like its not elegant. Its just my opinion. I don't mind leaving it